flag/service/ingresscontroller/ingress_controller.go

```diff
@@ -1,6 +0,0 @@
-package ingresscontroller
-
-type IngressController struct {
-	Namespace string
-	Service   string
-}
```

Comment:
Initially I thought the operator should be configured with relevant information. After thinking about how operators are supposed to work I think this is wrong. All information operators need to do their job must be given via the watched TPR. Operators should be stateless and only configured with resource specific information at runtime.